testsuite/features/step_definitions/navigation_steps.rb

```diff
@@ -876,3 +876,17 @@
 When(/^I scroll to the top of the page$/) do
   execute_script('window.scrollTo(0,0)')
 end
+
+# Navigate to a service endpoint
+When(/^I visit "([^"]*)" endpoint of this "([^"]*)"$/) do |service, host|
+  node = get_target(host)
+  port, text = case service
+                when 'Prometheus' then [9090, "graph"]
+                when 'Prometheus node exporter' then [9100, "Node Exporter"]
+                when 'Prometheus apache exporter' then [9117, "Apache Exporter"]
+                when 'Prometheus postgres exporter' then [9187, "Postgres Exporter"]
+                else raise "Unknown port for service #{service}"
+                end
+  _output, code = node.run("curl -s http://localhost:#{port} | grep -i '#{text}'")
```

Comment:
It might be better to assure that the endpoint can be requested outside localhost.